packages/material-ui/src/FormControl/FormControl.test.js

```diff
@@ -83,68 +89,76 @@ describe('<FormControl />', () => {
 
   describe('prop: disabled', () => {
     it('will be unfocused if it gets disabled', () => {
-      const wrapper = mount(<FormControl />);
-      setState(wrapper, { focused: true });
+      const inputRef = React.createRef();
+      const wrapper = mount(
+        <FormControl>
+          <Input ref={inputRef} />
+        </FormControl>,
+      );
+
+      inputRef.current.focus();
       wrapper.setProps({ disabled: true });
-      assert.strictEqual(getState(wrapper).focused, false);
+      wrapper.setProps({
```

Comment:
With the spy approach from above I would mount this immediately instead of inserting it later. This way you can do before/after assertions.